src/applications/personalization/profile-2/components/MilitaryInformation.jsx

```diff
@@ -1,5 +1,182 @@
+                  Get instructions from our help center
+                </a>
+                .
+              </div>
+            }
+          />
+        );
+      } else if (some(error.errors, e => ['500', '503'].includes(e.code))) {
+        return <LoadFail information="military" />;
+      }
+    }
+
+    if (serviceHistory.length === 0) {
+      return (
+        <AlertBox
+          isVisible
+          status="warning"
+          headline="We can’t access your military information"
+          content={
+            <p>
+              We’re sorry. We can’t access your military service records. If you
+              think you should be able to view your service information here,
+              please file a request to change or correct your DD214 or other
+              military records.
+            </p>
+          }
+        />
+      );
+    }
+
+    return (
+      <table className="militaryInformation" data-field-name="serviceHistory">
+        <thead>
+          <tr>
+            <th colSpan="2">
+              <h3>Period of Service</h3>
+            </th>
+          </tr>
+        </thead>
+        <tbody>
+          {serviceHistory.map((service, index) => (
+            <tr key={index}>
+              <td>
+                <span>
+                  <h4>{service.branchOfService}</h4>
+                </span>
+                <span>
+                  {moment(service.beginDate).format('MMM D, YYYY')} &ndash;{' '}
+                  {moment(service.endDate).format('MMM D, YYYY')}
+                </span>
+              </td>
+            </tr>
+          ))}
+        </tbody>
+      </table>
+    );
+  };
+
+  render() {
+    return (
+      <div>
+        <LoadingSection
+          isLoading={!this.props.militaryInformation}
+          message="Loading military information..."
+          render={this.renderContent}
+        />
+        <AdditionalInfo
+          triggerText="What if my military service information doesn't look right?"
+          onClick={() => {
+            recordEvent({
+              event: 'profile-navigation',
+              'profile-action': 'view-link',
+              'profile-section': 'update-military-information',
+            });
+          }}
+        >
+          <p>
+            Some Veterans have reported seeing military service information in
+            their VA.gov profiles that doesn’t seem right. When this happens,
+            it’s because there’s an error in the information we’re pulling into
+            VA.gov from the Defense Enrollment Eligibility Reporting System
+            (DEERS).
+          </p>
+          <p>
+            If the military service information in your profile doesn’t look
+            right, please call the Defense Manpower Data Center (DMDC). They’ll
+            work with you to update your information in DEERS.
+          </p>
+          <p>
+            To reach the DMDC, call 1-800-538-9552, Monday through Friday
```

Comment:
I know this is a copy/paste job, but might as well make the phone number a link. this is a good example of current best practices for a11y: https://github.com/department-of-veterans-affairs/vets-website/blob/8befb300dcc5e03000992af1e16792f679487b36/src/applications/personalization/profile360/components/PaymentInformationBlocked.jsx#L18-L23
although in this case the `aria-label` could be `aria-label="800. 5 3 8. 9 5 5 2."` with `800` as one word.